doc/about-sycl.rst

```diff
@@ -97,7 +101,7 @@ other approaches:
   (TLS) queues, its own scheduler, etc. atop the limited OpenCL_ stack to
   provide computation and communication overlap in a natural pain-free
   fashion. This relieves the programmer to reorganize her application to
-  work around these limitation, which can be quite a cumbersome work.
+  work around these limitations, which can be quite a cumbersome work.
```

Comment:
can be quite a cumbersome work -> can be quite cumbersome work

Reply:
I force-pushed the branch to fix this.